src/mon/MonCommands.h

```diff
@@ -314,7 +314,7 @@ COMMAND_WITH_FLAG("mds set_max_mds " \
 	"set max MDS index", "mds", "rw", "cli,rest", FLAG(OBSOLETE))
 COMMAND_WITH_FLAG("mds set " \
 	"name=var,type=CephChoices,strings=max_mds|max_file_size"
-	"|allow_new_snaps|inline_data|allow_multimds|allow_dirfrags " \
+	"|allow_new_snaps|inline_data|allow_dirfrags " \
```

Comment:
Missed `allow_dirfrag` here.
 
I think we should add an advanced config option to disable creating new dirfrags in the event of unforseen bugs.

Reply:
Actually, I forgot to re-add multimds; we need to keep that here if we are going to deprecate the flag rather than remove it.

Reply:
Ah right.